humble/distribution.yaml

```diff
@@ -705,6 +705,24 @@ repositories:
       url: https://github.com/flynneva/bno055.git
       version: main
     status: maintained
+  boeing_gazebo_model_attachment_plugin:
+    doc:
+      type: git
+      url: https://github.com/Boeing/gazebo_model_attachment_plugin.git
+      version: humble
+    source:
+      type: git
+      url: https://github.com/Boeing/gazebo_model_attachment_plugin.git
```

Comment:
So the big problem with this is the modified license on the repository.  Because it is not one of the standard OSI-approved licenses, we can't say that we are allowed to redistribute the results, and we are not lawyers.

I think the only ways forward here are to change this to an OSI-approved license (i.e. plain Apache-2.0), or close this PR.

Reply:
We will investigate changing this to plain Apache-2.0.

Reply:
@RoBeau friendly ping

Reply:
Hi, we have been given verbal approval to change to the plain Apache-2.0 license. Just waiting for final sign off in the next week or so.

Reply:
@clalancette thanks for you patients, I have updated both packages to use plain Apache-2.0.